xml/System.Text.Json.Serialization/JsonNumberHandling.xml

```diff
@@ -40,7 +40,8 @@
       </ReturnValue>
       <MemberValue>4</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The "NaN", "Infinity", and "-Infinity" <see cref="F:System.Text.Json.JsonTokenType.String" /> tokens can be read as floating-point constants, and the <see cref="T:System.Single" /> and <see cref="T:System.Double" /> values for these constants will be written as their corresponding JSON string representations.</summary>
```

Comment:
@Jozkee Not sure if there are enum/token/types for `Nan, Infinity and -Infinity`? If there are, consider using the enum itself with a `<see cref="Some Enum" />`

Reply:
@Jozkee @layomia @steveharter can address this later.